wscript

```diff
@@ -767,11 +777,18 @@ video_output_features = [
             check_cc(lib="EGL"),
             check_cc(lib="GLESv2"),
         ),
-    } , {
+    }, {
+         'name': '--rkmpp',
+         'desc': 'RockChip MPP  support',
+         'func': compose_checks(
+                 check_pkg_config('rockchip_mpp'),
+                 check_cc(lib="GLESv2")
+                 ),
+    }, {
```

Comment:
Putting here some comments @wm4 made in my repo : This should be removed as mpv doesn't use any symbols from MPP. the HWDEC_RKMPP Ifdef in vd_lavc can be removed as well.